rmgpy/qm/gaussian.py

```diff
@@ -242,8 +242,9 @@ def generateQMData(self):
         """
         Calculate the QM data and return a QMData object.
         """
+        #still can't handle charged atoms for QMData
         for atom in self.molecule.vertices:
-            if atom.atomType.label in ('N5s', 'N5d', 'N5dd', 'N5t', 'N5b'):
+            if atom.charge !=0:
```

Comment:
70% sure the answer is "yes". I think we were previously preventing pentavalent nitrogen because they always come with a charge. Although I can't quite remember off the top of my head, my guess is that I changed it because it started failing on some of the new charged atoms. 

I will investigate a bit further.